website/docs/docs/collaborate/project-recommendations.md

```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
 description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
 ---
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 
 
-Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing. 
-
-These recommendations provide insight into how you can build a more well documented, well tested, and well built project, leading to less confusion and more trust.
+- Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing.
+- These recommendations provide insight into how you can create a better-documented, better-tested, and better-built dbt project, leading to less confusion and more trust.
+- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project.
```

Comment:
```suggestion
- For a seamless and consistent experience, recommendations uses ` dbt_project_evaluator`'s pre-defined settings and don't import customizations applied to your package or project.
```

Reply:
Hiya @mirnawong1, I think the previous sentence was really effective and well explained. It flowed really nicely.
```suggestion
- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project..
```

Reply:
Thanks @nataliefiann ! Do  you m we a the sentence that doesn't mention project evaluator? I see what you mean and i wanted to clarify or be more explicit in the exact predefined a settings that recommendation uses. I thought previously it wasn't as clear